games-fps/prboom-plus/prboom-plus-2.5.1.4.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools desktop xdg
+
+DESCRIPTION="A Doom source port developed from the original PrBoom project"
+HOMEPAGE="http://prboom-plus.sourceforge.net"
+# We are using a github mirror here because the upstream tarball is missing the
+# free dog assets; we should update SRC_URI to point to the upstream tarball
+# once the free assets are present as expected
+#SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+SRC_URI="https://github.com/coelckers/prboom-plus/archive/f96f891d068dcc5ec52ed91056b46d27e9a8462d.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+
+	nonfree? ( freedist )
+	!nonfree? ( BSD )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dumb fluidsynth +mad net nonfree +opengl pcre +png portmidi sdl-image +sdl-mixer +vorbis"
+REQUIRED_USE="sdl-image? ( opengl )"
+
+DEPEND="
+	media-libs/libsdl
+	dumb? ( media-libs/dumb )
+	fluidsynth? ( media-sound/fluidsynth )
```

Comment:
Please check whether this works with fluidsynth-2 and adjust accordingly.

Reply:
The code was written before fluidsynth 2.0.0 was released so it naturally expects fluidsynth-1. However, running with fluidsynth-2 does appear to work: I can hear the iconic "At Doom's Gate" music track play when I start the first level.

There are several error messages which are printed when I run with fluidsynth-2 (not unexpected given the API change), but they are not showstoppers and appear to be related to configuration of the MIDI player.

I don't believe it is wise to restrict the dependency to only fluidsynth-2 just yet, since I'm not sure what sort of errors may appear later on. If there are no errors, then fluidsynth-1 will be eventually dropped from Gentoo anyway; if there are serious errors, then we can just drop the `fluidsynth` USE flag and have users rely on the several other MIDI player options.

Reply:
That's fine, it just need `:=` then. I'll add that.